run/sigterm-handler/main.go

```diff
@@ -12,9 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Sample run-sigterm shows how to gracefully shut down in response to a SIGTERM signal.
+// Sample sigterm-handler shows how to gracefully shut down in response to a SIGTERM signal.
 package main
 
+// [START cloudrun_sigterm_handler]
```

Comment:
Please move above the `package main` and corresponding comment. That way, users can copy and paste the entire file.

Reply:
The [snippet guidelines](https://docs.google.com/document/d/1cPBH98ay6fzZU0l1JHTj46J5y1e2KvYtY8Iv33Ov-zQ/edit) say not to include the `package main`. @adamross do you think this should be a snippet or show the full file?

Reply:
Here is the Go samples style guide: https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#include-imports-and-flags-in-region-tags

Hosting samples aren't necessarily quickstarts, but I think the `package main` logic still applies.

The existing Cloud Run samples include the package declaration:
https://github.com/GoogleCloudPlatform/golang-samples/blob/a6135c8a474c3a4c11eccb40814bb9c966c6228c/run/helloworld/main.go#L15-L19

Reply:
This is a snippet for a how-to, whereas the others were "slices of a sample app in a tutorial". I have no opinion either way, but I figure if the code isn't copy-and-paste usable (because we're excluding the handler) that leaving out the package declaration is reasonable.